src/test/java/jenkins/branch/OrganizationFolderTest.java

```diff
@@ -90,4 +116,30 @@ public String getDisplayName() {
         }
     }
 
+    private RingBufferLogHandler createJULTestHandler() throws SecurityException, IOException {
+        RingBufferLogHandler handler = new RingBufferLogHandler();
+        SimpleFormatter formatter = new SimpleFormatter();
+        handler.setFormatter(formatter);
+        Logger logger = Logger.getLogger(MultiBranchImpl.class.getName());
+        logger.addHandler(handler);
+        return handler;
+    }
+
+    private void waitForLogFileMessage(String string, RingBufferLogHandler logs) throws IOException, InterruptedException {
+        File rootDir = r.jenkins.getRootDir();
+        synchronized (rootDir) {
+            int limit = 0;
+            while (limit < 5) {
+                rootDir.wait(1000);
```

Comment:
Nobody, it just wait for 1 second (5 in total) for the message to appear, then returns or assert unconditionally.